frontend/src/scenes/exports/ExportsList.tsx

```diff
@@ -13,6 +13,121 @@ export const scene: SceneExport = {
     component: Exports,
 }
 
+export interface ExportActionButtonsProps {
+    currentTeamId: number
+    export_: BatchExport
+    loading: boolean
+    updateCallback: (signal: AbortSignal | undefined) => void
+}
+
+export function ExportActionButtons({
+    currentTeamId,
+    export_,
+    loading,
+    updateCallback,
+}: ExportActionButtonsProps): JSX.Element {
+    const { executeExportAction: pauseExport, error: pauseError } = useExportAction(currentTeamId, export_.id, 'pause')
+    const { executeExportAction: resumeExport, error: resumeError } = useExportAction(
+        currentTeamId,
+        export_.id,
+        'unpause'
+    )
+
+    const { deleteExport, error: deleteError } = useDeleteExport(currentTeamId, export_.id)
+
+    return (
+        <div className={clsx('flex flex-wrap gap-2')}>
+            <LemonButton
+                status="primary"
+                type="secondary"
+                onClick={() => {
+                    export_.paused
+                        ? resumeExport().then(() => {
+                              if (resumeError === null) {
```

Comment:
I'm not 100% sure how this works in terms of state changes, rerenders and closures.  It would be more correct in my opinion to do something like `resumeExport().then(...show the success toast...).catch(...show the failure toast...)`

I _think_ the error toast would be unpredictable atm but I could be wrong.

Reply:
Or something like `resumeExport().then((response) => ... show toast based on response ...)`

Reply:
I've updated this code to use `then(...).catch(...)` instead of `if (error === null) {...} else {...}`